### PR TITLE
#166178179 Fetch all Books from db for UI

### PR DESCRIPTION
**Pull request by tolumide-ng**
#### What does this PR do?
Fetches data for all available books and displays on the server

#### Description of Task to be completed?
- Adds Price column in books table to store the cost to rent a book
- Modifies book generator on utils to generate fake price for test
- Modifies seed files to include fake price with faker component
- User can view all books on the database
- User can get the total number of pages and view them

#### How should this be manually tested?
- Clone the repo 
- Cd into the repo
- Produce a local .env file
- Install all dependencies with **npm install** or **yarn install**
- Run seed with **npm run seed** or **yarn run seed**
- Start the server with **npm run dev** or **yarn run dev**
- Visit **localhost/{PORT}/books**

#### Any background context you want to provide?
image

#### What are the relevant pivotal tracker stories?
#166178179

### CODE REVIEW

File: client/styles/main.css [hunk 1]
```diff
@@ -23,3 +23,9 @@
 .w-72 {
     width: 18rem;
 }
+
+.ellipsis {
```
Comment (bahdcoder) [inline client/styles/main.css]: Why does this not follow the convention ?

File: client/pages/Books/index.js [hunk 2]
```diff
@@ -1,49 +1,96 @@
...
+        return finishedRow;
+    }
+
+    const fetchData = async () => {
+        try {
+            const response = await axios.get(url);
+            setData(response.data.data.results)
+            setTotalPages(response.data.data.total);
+            setIsLoading(false);
+        }catch(err){return err}
```
[the remark this answers is not in the document]
Comment (bahdcoder) [reply client/pages/Books/index.js]: For now you can allow this. We'll have a way to handle this using toast messages.

File: client/pages/Books/index.js [hunk 3]
```diff
@@ -1,49 +1,96 @@
...
+                            <div className='border-t-2 border-black w-3 mb-1' />
+                        </div>
+                        <p className='ml-2 font-raleway'>Filter</p>
+                    </button>
+                    <input type='text' placeholder='search for books' 
+                    className='bg-gray-200 rounded-full w-full p-2 text-black-1000
+                    focus:rounded-full pl-6 font-semibold text-sm font-raleway bg-light-gray' />
+                </div>
+                <div>
+                    {isLoading ? console.log('...waiting'): loadBooks(data)}
```
[the remark this answers is not in the document]
Comment (tolumide-ng) [reply client/pages/Books/index.js]: Removed

File: client/pages/Books/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (tolumide-ng) [reply client/pages/Books/index.js]: I'd try to readup on that

File: client/styles/main.css (hunk 1 above)
Comment (tolumide-ng) [reply client/styles/main.css]: I no longer need the ellipsis, I would be removing it

**Pull request merged**